compiler/circle2circle-dredd-recipe-test/test.lst

```diff
@@ -141,3 +141,5 @@ Add(REGRESS_ONNX_Conv_BN_Relu6_001 PASS
 
 Add(REGRESS_ONNX_Mul_Mul_000 PASS
       convert_nchw_to_nhwc)
+
+Add(RemoveDeadNodeWithQueryPass_000 PASS substitute_pack_to_reshape)
```

Comment:
Maybe you should change test title

Reply:
I’m not sure if this title is correct.

Reply:
I think `REGRESS_Issue_13863` would be ok.